### PR TITLE
 modifying amb_scale and unamb_scale parameter names

### LINKED ISSUE #982: Coherence between ambig fname and scale
## **Desired feature/enhancement**

It would be nice if `_fname` and `_scale` parameters were staring with the same prefix for all parameters.
It is already the case for `hbond`, but not for `ambig` and `unambig`

## **Motivation**
Make it more coherent

### PR DESCRIPTION
**Pull request by VGPReys**
You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and that you comply with the following criteria:

- [ ] You have sticked to Python. Please talk to us before adding other programming languages to HADDOCK3
- [ ] Your PR is about CNS
- [ ] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
- [ ] You structured the code into small functions as much as possible. You can use classes if there is a (state) purpose
- [ ] Your code follows our coding style
- [ ] You wrote tests for the new code
- [ ] `tox` tests pass. *Run `tox` command inside the repository folder*
- [ ] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
- [ ] PR does not add any dependencies, unless permission granted by the HADDOCK team
- [ ] PR does not break licensing
- [ ] Your PR is about writing documentation for already existing code :fire:
- [ ] Your PR is about writing tests for already existing code :godmode:

---

Closes #982 

Modifies parameters `amb_scale` to `ambig_scale` and `unamb_scale` to `unambig_scale`

### CODE REVIEW

File: src/haddock/modules/refinement/flexref/cns/flexref.cns [hunk 1]
```diff
@@ -321,12 +321,12 @@ evaluate ($failure = $result)
 
 
 {* Define initial weights for distance restraints ============================== *}
-evaluate ($unamb_scale = $Data.unamb_hot)
+evaluate ($unambig_scale = $Data.unamb_hot)
```
Comment (amjjbonvin) [inline src/haddock/modules/refinement/flexref/cns/flexref.cns:324]: shouldn't then also unamb_hot, unamb_cool... and related be modified for consistency?

**Review by amjjbonvin — CHANGES_REQUESTED** [review]
There are other parameters than should be changed for consistency, related to the simulated annealing hot and cool stages

File: src/haddock/modules/refinement/flexref/cns/flexref.cns (hunk 1 above)
Comment (VGPReys) [reply src/haddock/modules/refinement/flexref/cns/flexref.cns:324]: extremely good point here !

**Comment by VGPReys** [comment]
Also, we do not have any example tuning those parameters (`_scale`, `_hot`, `cool1`, `cool2` nor `cool3`)

**Comment by rvhonorato** [comment]
You don't need an example for this, that's what the integration tests are for - just add one that uses these parameters

**Comment by amjjbonvin** [comment]
We should not.

**Review by amjjbonvin — APPROVED** [review]

**Review by amjjbonvin — APPROVED** [review]

**Review by rvhonorato — APPROVED** [review]

**Pull request merged**